src/main/java/com/paneedah/weaponlib/PlayerRenderer.java

```diff
@@ -13,16 +12,16 @@
 import org.lwjgl.opengl.GL11;
 
 import java.nio.FloatBuffer;
-import java.util.List;
 
 public class PlayerRenderer {
```

Comment:
Most of the changes to this class seems uncalled for and not related to this pr goals